pkg/cmd/flags/filter.go

```diff
@@ -88,38 +89,7 @@ To 'escape' those operators, please use single quotes, e.g.: 'uid>0'
 `
 }
 
-func PrepareFilter(filtersArr []string) (tracee.Filter, error) {
```

Comment:
Don't forget to add examples in the `filterHelp()` string output AND to explain what it is.